File: RootNpcGenerator/RootNpcBackend/Models/Npc.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace RootNpcBackend.Models
{
    public class Npc
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }
        public string Name { get; set; }
        public Race Race { get; set; }
        public Age Age { get; set; }
        public Gender Gender { get; set; }
        public Faction Faction { get; set; }
        public Weapon Weapon { get; set; }
        public Armor Armor { get; set; }
        public int Injury { get; set; }
        public int Exhaustion { get; set; }
        public int Moral{ get; set; }
      

    }
}

File: RootNpcGenerator/RootNpcLara/app/Models/Npc.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Npc extends Model
{
    use HasFactory;

    protected $fillable = [
        'name',
        'race',
        'age',
        'gender',
        'faction',
        'weapon',
        'armor',
        'injury',
        'exhaustion',
        'moral'
    ];

    public function __construct(array $attributes = array())
    {
        parent::__construct($attributes);
    }
}
